static_src/components/users.jsx

```diff
@@ -159,6 +159,11 @@ export default class Users extends React.Component {
                 onRemovePermissions={ this.handleRemovePermissions }
             />
           </div>
+          <p>For more information on different roles and what they mean see <a
```

Comment:
@msecret what do you think about putting this at the top of the page for now? I think its worthwhile context and its hard to find if there are a decent number of users in an org.